DOC-3329: Premium Plugins Installation and Bundling Documentation for NPM (#3928)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/language_url.adoc

----
tinymce.init({
  selector: 'textarea', // change this value according to your HTML
  language_url: '/<path_to_language_pack>/fi.js', // path from the root of your web application — / — to the language pack(s)
  language: 'fi'  // the name of the language pack file
});
----

You can find and download languages link:{gettiny}/language-packages/[here].

=== Example: loading premium languages from `+tinymce-premium+` (NPM)

> When installing the `+tinymce-premium+` NPM package, each paid translation is stored in `+node_modules/tinymce-premium/langs/+`. Configure `+language_url+` to point at the desired file.

Reviewer: ```suggestion
When installing the `+tinymce-premium+` NPM package, each translation is stored in `+node_modules/tinymce-premium/langs/+`. Configure `+language_url+` to point at the desired file.
```

.Example
[source,js]
----
tinymce.init({
  selector: 'textarea',
  language_url: '/<path_to_tinymce-premium>/langs/pt_BR.js',
  language: 'pt-BR'
});
----

[NOTE]
====

> * Ensure the `+node_modules+` directory (or a copy of the language file) is accessible to the browser. Many projects copy the required file into a public folder (such as `+/public/langs/pt_BR.js+`) during their build step and update `+language_url+` accordingly.

Reviewer: ```suggestion
* Ensure the language file is accessible in the application. Many projects copy the required file into a static assets folder (such as `+/public/langs/pt_BR.js+`) during their build step.
```
